NosTests/Model/AuthorTests.swift

```diff
@@ -226,26 +226,27 @@ final class AuthorTests: CoreDataTestCase {
         // Assert
         XCTAssertEqual(events.count, 0)
     }
-
-    // MARK: - Helpers
-
-    private func createTestEvent(
-        in context: NSManagedObjectContext,
-        publicKey: RawAuthorID = KeyFixture.pubKeyHex,
-        deletedOn: Set<Relay> = []
-    ) throws -> Event {
-        let event = Event(context: context)
-        event.createdAt = Date(timeIntervalSince1970: TimeInterval(1_675_264_762))
-        event.content = "Testing nos #[0]"
-        event.deletedOn = deletedOn
-        event.kind = 1
-
-        let author = Author(context: context)
-        author.hexadecimalPublicKey = publicKey
-        event.author = author
-
-        let tags = [["p", "d0a1ffb8761b974cec4a3be8cbcb2e96a7090dcf465ffeac839aa4ca20c9a59e"]]
-        event.allTags = tags as NSObject
-        return event
+    
+    func test_outOfNetwork_givenCircleOfFollows() throws {
```

Comment:
This is a little bonus test I threw in because I had a hunch that the cleanup function was deleting too many Follows or Authors, so I wrote a quick test for that. It looks like it is functioning as intended though.